<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="WIN-QNH2QLCL96P" failures="2" tests="2" name="Runner.TestRunner" time="11.386" errors="0" timestamp="2024-01-01T16:44:51 EET" skipped="0">
  <testcase classname="Runner.TestRunner" name="runScenario" time="0.408">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at Steps.UserUpdateWarehouseData.the_user_at_saas_login_page(UserUpdateWarehouseData.java:43)
at ✽.The User at SAAS Login Page(file:///C:/Users/Eslam/eclipse-workspace/TAF/src/test/java/Features/userUpdateWarehouseData.feature:5)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="Runner.TestRunner" name="runScenario" time="10.978">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at Steps.UserUpdateWarehouseData.check_that_the_warehouse_data_update_is_completed_depend_on_the_test_condation(UserUpdateWarehouseData.java:91)
at ✽.Check That The Warehouse Data Update Is Completed Depend On The Test Condation(file:///C:/Users/Eslam/eclipse-workspace/TAF/src/test/java/Features/userUpdateWarehouseData.feature:12)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- Runner.TestRunner -->
